Uppsala/UppEt12.xml

```diff
@@ -0,0 +1,885 @@
+                                        <bibl>
+                                            <ptr target="bm:BmWebsite"/>
+                                        </bibl>
+                                    </listBibl>
+                                    <listBibl type="secondary">
+                                        <bibl>
+                                            <ptr target="bm:PEMM"/><citedRange unit="item">L-Uppsala (UU) 30</citedRange>                                        
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Bausi2015Wansleben"/>
+                                            <citedRange unit="page">214</citedRange>
+                                            <citedRange unit="number">58</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Bausi1989MssBNCF"/>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Cerulli1956Upsala"/>
+                                            <citedRange unit="page">170–178</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    
+                </msDesc>
+
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+            <tagsDecl>
+                <rendition xml:id="partialRubric">Red ink is used partially in the rubrication of punctuation and numerals.</rendition>
+            </tagsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage><language ident="en">English</language></langUsage>
+            <langUsage><language ident="gez">Gəʿəz</language></langUsage>
+            <langUsage><language ident="de">German</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2025-02-06">Created XML record</change>
+        </revisionDesc>
+    </teiHeader>
+    <facsimile>
+        <graphic url="http://urn.kb.se/resolve?urn=urn:nbn:se:alvin:portal:record-510878"/>
+    </facsimile>
+    <text xml:base="https://betamasaheft.eu/">
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:hasUsed" active="PRS10036Wansleb" passive="UppEt12"/>
```

Comment:
saws:hasUsed should only be used with works

Reply:
Zhenia and I talked about this in another pull request and then I think we concluded that it was fine to use it with persons as well

Reply:
Could you update the schema then, please?

Reply:
Do you mean the schema or the guidelines?

Reply:
Both, assuming that https://betamasaheft.eu/Guidelines/?id=relation is not synced to the schema

Reply:
@eu-genia , based on what you commented before (https://github.com/BetaMasaheft/Manuscripts/pull/2801#discussion_r1933116157), do you agree that the schema and guidelines can be updated?